src/DataCollection.WPF/Views/IdentifiedFeaturePopup.xaml

```diff
@@ -149,16 +149,18 @@
                                                     <TextBlock Text="{Binding DisplayedFields[2].FormattedValue}" FontSize="12" Foreground="Gray" Margin="7,0,0,10"/>
                                                 </StackPanel>
                                                 <Button Content="&#xE76C;" FontFamily="Segoe MDL2 Assets" FontSize="15" Width="30" Height="30" VerticalAlignment="Center" Style="{StaticResource MenuButtonStyle}" 
-                                                        Grid.Column="1" Grid.Row="1" HorizontalAlignment="Right" Margin="0,0,5,0" 
-                                                        ToolTip="{Binding ., Converter={StaticResource LocalizationConverter}, ConverterParameter=ShowDetailButton_Tooltip}"
-                                                        Command="{Binding DataContext.IdentifiedFeatureViewModel.SetSelectedOriginRelationshipCommand, RelativeSource={RelativeSource AncestorType=UserControl}}"
-                                                        CommandParameter="{Binding}">
+                                                    Grid.Column="1" Grid.Row="1" HorizontalAlignment="Right" Margin="0,0,5,0" 
```

Comment:
The XAML files are starting to get a little unwieldy, especially with line wrapping turned on. Not for this PR, but it would be worthwhile doing an automated reformatting to make the XAML easier to read in VS.

Reply:
Oh, that's a great idea! I just did that and it looks so much better. Thanks for the suggestion!